src/reducers/map/create/success.ts

```diff
@@ -14,31 +14,15 @@ var invariantArgs: InvariantsBaseArgs = {
 export default function success(config: Config, current: Map<any>, addedRecord: any, clientGenKey?: string): Map<any> {
 	invariants(invariantArgs, config, current, addedRecord)
 
-	var key = config.key
-	var done = false
-	var addedRecordKey = addedRecord[key]
-
-	// Update existing records
-	var updatedCollection = r.map((existingRecord) => {
-		var recordKey = existingRecord[key]
-		if (recordKey == null) throw new Error('Expected record to have ' + key)
```

Comment:
@sporto Seemed weird to me to check here that previously inserted records have keys, maybe this should be added to the create start reducer instead?